Note tested / supported versions of Redis in the Readme

lewispb opened the pull request:
Relates to https://github.com/rails/kredis/issues/77

dhh commented:
On second reflection, it's a bit of a weak guarantee, this? "tested on". Maybe we should just find out where it doesn't work, and document the minimum viable version?

lewispb commented:
Yeah, thinking of putting together a CI matrix of Redis clients + servers, if that sounds good?

lewispb commented:
Redis server version 2 and 3 cause failures in the test suite, so we can say that for 100% of Kredis functionality to work, Redis server version 4 onwards is required.

I've updated the Readme to reflect this, and added a CI matrix for Redis server versions (if you could approve the workflow, that'd be great).
